tracer/test/Datadog.Trace.Tests/TraceContextTests.cs

```diff
@@ -168,9 +166,9 @@ public void FullFlushShouldNotPropagateSamplingPriority()
 
             spans.Value.Should().NotBeNullOrEmpty("a full flush should have been triggered");
 
-            rootSpan.GetMetric(Metrics.SamplingPriority).Should().Be(SamplingPriorityValues.UserKeep, "priority should be assigned to the root span");
+            rootSpan.GetMetric(Metrics.SamplingPriority).Should().Be(null, "because sampling priority is not added until serialization");
```

Comment:
nit: when the assert fails and prints out a message, does it already add the word "because"? If so, the "because" in the string will be redundant and hard to read

Reply:
Nice catch 😝 I noticed recently that if you don't include "because", the testing library will add it. But if you _do_ include it, the library won't add it a second time!

Known this, I decided that I prefer including it because the code reads more like English:

```csharp
value.Should().BeNull("because sampling priority is not added until serialization");
```
read: "value should be null _because_ sampling priority is..."

And without the explicit "because", my brain assumes the string is the just the error message. I've seen a few instances in our code where other devs thought the same thing. Something like
```csharp
value.Should().BeNull("Error doing something!");
```

…which results in the message, "Expected `value` to be null because Error doing something!", which is silly.

Reply:
self-nit: use `Should().BeNull()`, not `Should().Be(null)`

Reply:
I learned about "because" from the example in https://fluentassertions.com/introduction, so it seems like the recommended way to do it, too.